### DISCUSSION TITLE
How to calcluate the volume of DCI-P3 when the luminance is 1000nits?

### DISCUSSION THREAD

**Discussion by Wangld5** [colour-science/colour#1010] [Q&A]
Hi, I want to know how to set the value of luminance of the colorspace and calculate the volume of it. For example, how to calculate the volume of DCI-P3 colorspace when the luminance is 1000nit. Thank you.

**Comment by thomasmansencal** [comment]
Hello @Wangld5,

If you are working with relative luminance values and compute the value for the colourspace by default and you assume that 1 is 100 nits, then the volume at 1000 nits should be the volume at 100 nits multiplied by 10^3?

Cheers,

Thomas

**Reply by Wangld5** [reply]
I have tried this way, but the volume did not change even I multiplied by 10^3.

**Reply by thomasmansencal** [reply]
How did you compute the volume?
